grails 5 upgrade

niravassar opened the pull request.

Commit:
grails 5 upgrade

sdelamo commented on complete/build.gradle:
```diff
@@ -6,9 +6,8 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails version

sdelamo commented on complete/build.gradle:
```diff
@@ -85,15 +76,6 @@ bootRun {
-}
-
 tasks.withType(Test) {
     systemProperty "geb.env", System.getProperty('geb.env')
```
if we are not using geb, we can remove the geb system properties configurations.

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

sdelamo commented on initial/build.gradle:
```diff
@@ -1,29 +1,28 @@
         maven { url "https://repo.grails.org/grails/core" }
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
```
this should be decoupled from grails version

sdelamo commented on initial/build.gradle:
```diff
@@ -84,15 +76,6 @@ bootRun {
-}
-
 tasks.withType(Test) {
     systemProperty "geb.env", System.getProperty('geb.env')
```
get rid of Geb related configuration if we are not using Geb.

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on initial/gradle.properties:
```diff
@@ -1,2 +1,6 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
grails version should be 5.0.1

jamesfredley committed:
Modernize CI to Java 21 + Gradle 9 baseline (refs apache/grails-static-website#354)

jamesfredley committed:
Revert "Modernize CI to Java 21 + Gradle 9 baseline (refs apache/grails-static-website#354)"

This reverts commit 90be8904e902f37ea5e73b464879c3d17da0dabe.

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
